chakra-ui/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "chakra-ui-meteor-example",
+  "private": true,
+  "scripts": {
+    "start": "meteor run"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.11.2",
+    "@chakra-ui/icons": "^1.0.6",
+    "@chakra-ui/react": "^1.3.4",
+    "@emotion/react": "^11.1.5",
+    "@emotion/styled": "^11.1.5",
+    "framer-motion": "^3.10.0",
+    "meteor-node-stubs": "^1.0.1",
+    "react": "^16.13.1",
```

Comment:
Make sure all the dependencies are up-to-date, for example, React is not.

Reply:
All right. I just updated React to the latest version.